notebooks/7-rotation-speeds/interm_rotation_rate_solutions.ipynb

```diff
@@ -0,0 +1,3039 @@
+{
```

Comment:
<p>Just delightful. This plot reminds me of a stringed musical instrument.</p><p> </p><p>I think the axis labels are backward, though; I should hope none of them have a period ratio of 0</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>